lib/airbrake/rake/tasks.rb

```diff
@@ -106,8 +106,19 @@
            " environment will be used."
     end
 
+    unless (repo = ENV['REPOSITORY_URL'])
+      repo = `git remote get-url origin 2>/dev/null`.chomp
+      if repo.empty?
+        puts "Airbrake couldn't identify your app's repository."
```

Comment:
I think this message can be more concrete: ``Airbrake couldn't identify `origin` of your app's repository`` (or something similar). This way it's obvious why it couldn't detect the repo.